### DISCUSSION TITLE
link to terraform-provider releases links to incorrect version information page

### DISCUSSION THREAD

**Discussion by ppuschmann** [renovatebot/renovate#32344] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

We're using renovate on some repos with Terraform code and everything is working out.

The only thing that is weird is, that the link in the MR points to some page that is not showing the latest release page.

Is this an issue of renovate or of the release-management of the actual terraform-provider we're  using?

Here: a MR for the google-provider in v6.10.0 is created, the link points to https://github.com/hashicorp/terraform-provider-google/blob/HEAD/CHANGELOG.md#6100-Unreleased

--> unfortunately the link points to a page where v6.10.0 is apparently still *unreleased*.

This is not true, because the original GitHub releases link is:
https://github.com/hashicorp/terraform-provider-google/releases/tag/v6.10.0 which shows the correct information.

Do you have any idea about this?

**Comment by secustor** [comment, marked as answer]
Changelog.md files are taking precedence as these show more user friendly changelogs. 

Looking at the commit history of the file, they do not update this file automatically and have done this 2 days after your post. 
https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md

So right now it should link to the correct header.

**Reply by ppuschmann** [reply]
So basically this special release-management of `terraform-provider-google` is borken, okay I get that. 

I had only the idea, that we could use a different link in renovate, but apparently only the information was missing. 
Thank you for the kind answer.
